main.go

```diff
@@ -1,16 +1,69 @@
 package main
 
 import (
+	"encoding/json"
 	"log"
+	"net/http"
 	"os"
 
 	"github.com/apex/gateway"
+	"github.com/gin-contrib/static"
 	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
 )
 
+type thumbnailRequest struct {
+	Name 				string `json:"name"`
+	DesignationValue 	string `json:"designation_value"`
+	Business 			string `json:"business"`
+	CurrentWebsiteUrl 	string `json:"current_website_url"`
+	HelpValue 			string `json:"help_value"`
+	Reason 				string `json:"reason"`
+	Email 				string `json:"email"`
+	Message 			string `json:"message"`
+}
+
+func contactDetail(c *gin.Context) {
+	var input thumbnailRequest
+
+	err := c.ShouldBindWith(&input, binding.JSON)
+	if err != nil {
+		c.Abort()
+		c.JSON(http.StatusBadRequest, gin.H{"message": err.Error()})
+		return
+	}
+
+	err = json.NewDecoder(c.Request.Body).Decode(&input)
```

Comment:
API is working for me, This code will not require. The decoding request body is useless as ShouldBindWith is also doing the same